### DISCUSSION TITLE
GO terms for Nicotiana benthamiana

### DISCUSSION THREAD

**Discussion by crobinson12** [geneontology/helpdesk#496] [Q&A] [answered]
Hello,
I'm trying to find a .gz file with the GO terms for N.benthamiana, but I can't find it. Can anyone help me with this please? I've tried looking for it in the NCBI website, in the FTP files tab. 
Thank you!

**Comment by suzialeksander** [comment, marked as answer]
Hi,

Thank you for contacting the GO Helpdesk. Currently, NCBI does not have plants, but you can find a GAF anytime at QuickGO. From the homepage at [ https://www.ebi.ac.uk/QuickGO/](https://www.ebi.ac.uk/QuickGO/), click the GO Annotations square, then under Taxon enter 4100 for Nicotiana benthamiana. Click Add, then click Apply. 

Currently there are 7667 annotations, and I've attached these in GAF format. They're also available as TSV and GPAD. These annotations will be from the [2024-06-17/](http://release.geneontology.org/2024-06-17/index.html) GO Release.


[QuickGO-annotations-benthamiana-20240802.gaf.gz](https://github.com/user-attachments/files/16475915/QuickGO-annotations-benthamiana-20240802.gaf.gz)

Note the file format is GAF, but is zipped to allow sharing on GitHub.
